test/CMakeLists.txt

```diff
@@ -11,4 +11,8 @@ foreach(test ${TEST_SOURCES})
     set_target_properties(${testname} PROPERTIES CXX_STANDARD 20 CXX_EXTENSIONS OFF LINKER_LANGUAGE CXX)
     target_link_libraries(${testname} PRIVATE rpp)
     target_include_directories(${testname} PRIVATE ${RPP_INCLUDE_DIRS})
+
+    if (CMAKE_SYSTEM_PROCESSOR MATCHES "(x86)|(X86)|(amd64)|(AMD64)")
```

Comment:
The flag can also be removed here, since optimizing the tests doesn't matter

Reply:
True, it just prints an ugly warning when compiling the tests. But can probably be cleaned up with `march=native` or similar.

Reply:
Yeah - it should go away if we delete f32x8